acceptance-tests/src/test/scala/ch/renku/acceptancetests/workflows/Collaboration.scala

```diff
@@ -62,4 +65,47 @@ trait Collaboration {
     projectPage.Collaboration.Issues.NewIssue.createIssueButton click;
     sleep(3 seconds)
   }
+
+  def verifyBranchWasAdded(implicit projectDetails: ProjectDetails): Unit = {
+    val projectPage = ProjectPage()
+    When("the user navigates to the Collaboration tab")
+    click on projectPage.Collaboration.tab
+    Then("they should see a 'Do you want to create a merge request for branch...' banner")
+    verify userCanSee projectPage.Collaboration.MergeRequests.futureMergeRequestBanner
+  }
+
+  def createNewMergeRequest(implicit projectDetails: ProjectDetails): Unit = {
+    implicit val projectPage = ProjectPage()
+    When("the user navigates to the Collaboration tab")
+    click on projectPage.Collaboration.tab
+    And("they navigate to the Merge Request sub tab")
+    click on projectPage.Collaboration.MergeRequests.tab
+    // Give some time for the tab change to take effect
+    sleep(4 second)
+    And("the user clicks on the 'Create Merge Request' button")
+    click on projectPage.Collaboration.MergeRequests.createMergeRequestButton
+    sleep(4 seconds)
+    And("they navigate to the MergeRequest sub tab")
+    click on projectPage.Collaboration.MergeRequests.tab
+    sleep(3 seconds)
+    Then("the new Merge Request should be displayed in the list")
+    val mrTitles = projectPage.Collaboration.MergeRequests.mergeRequestsTitle
```

Comment:
Maybe it the `mergeRequestsTitle` should be `mergeRequestsTitles` as it looks it returns a list or a set?